app.rb

```diff
@@ -10,49 +10,90 @@
+#   puts tenants[:name]
+# end
 
   # Print only apartments that are less then 700 in rent
-rent = each.sort_by |monthly_rent|
- puts rent[:monthly_rent]
-end
+  # Cheap_apartments equals the monthly_rent that will be selected in the apartments. Print the apartments with monthy rents of less than 700. Show the length of the cheap_apartments. Exit the program.
+# cheap_apartments = apartments.select do |monthly_rent|
+#  puts apartments[:monthly_rent] < 700
+#  cheap_apartments.lengths
+# end
 
   # Print only tenants that are over the age of 44
-tenants = tenants.select do |tenant|
-  apartment]:age > 44
-end
-puts tenants
+  # Older_tenants represents tentants with the apartment_id of 2 that will be selected by tenants. Print older_tenants.
+# older_tenants = tenants.select { |tenant| tenant[:apartment_id] = 2 }
+#   puts older_tenants
+#   puts "*" * 50
+#   puts tenants
+
+
   # Print only tenants that have an apartment id of 1
-  apartment_one = tenants.select do |tenant|
-    tenant[:apartment_id] == 1
-  end
-  puts apartment_1
+# apartment_one = tenants.select do |tenant|
+#   tenant[:apartment_id] == 1
+# end
+# puts apartment_1
 
   # Print all the tenants in order from youngest to oldest
-age_tenants = tenants.sort_by do |tenant|
-   tenant[:age]
-end
-puts age_tenants
+  #Call on Tenants to be sorted in a specific order by age. Do it. Print tentants by age.
+# age_tenants = tenants.sort_by do |tenant|
+#    tenant[:age]
+# end
+# puts age_tenants
 
   # Print the names of all the tenants alphabetically
-  tenants.sort_by do |name|
-    puts tenants[:name]
-  end
+# tenant_name = tenants.sort_by do |name|
+#   tenants[:name]
+# end
+# puts tenant_name
 
   ## More challenging
   # When printing tenants also print out the address that the tenant resides in.
-  tenants = each.do |tenant|
-    puts tenant[:name]
-    tenant_apartment = apartments.find {|apartment| apartment[:id] == tenant[:apartment_id]}
-    puts tenants_apartment[]:name]
-  end}
+# tenants = each.do |tenant|
+#   puts tenant
+# tenant_apartment = apartments.find {|apartment| apartment[:id] == tenant[:apartment_id]}
+#   puts tenants_apartment[]:name]
+# puts
+#
+# end}
+
+
 
   # When printing all apartments, under each apartment print all of its tenants
+
+
+  # - I should have a numbered interface for my application so that I can just type in a number to access different parts of my program.
+  # - I should be allowed an option to view all the apartments so that I can get an overview of my properties.
+  # - I should be allowed  an option view all the tenants so that I can get an overview of my tenants
+
+  # Inspect data.rb: What are the property types for both hashes? Data structure??
+input = gets.chomp
+
+  # Use input to inquire unit id to create and view a new array (.map).
+if input == "1"
+apartments.map do |unit|
+  puts unit
+  end
+end
+
+# Do the same thing for tenants.
+if input == "2"
+tenants.map do |person|
+  puts person
+  end
+end
```

Comment:
What you have here actually works alright, it would be nice to have a `puts` statement telling the user that they should enter a 1 or 2,